commands/autoresponse/add.js

```diff
@@ -27,6 +27,8 @@ module.exports = async (responses, message) => {
         return;
     }
 
+    content = content.replace(/^`(.*)`$/,(a,b) => b);
+
     let flags;
     if (type === 'regex') {
```

Comment:
What if `type` is 'REGEX'? You have to make your types checks case insensitive everywhere or just put
`type = type.toLowerCase()`
somewhere at the beginning